src/components/slices/faculty-grid.module.scss

```diff
@@ -1,107 +1,176 @@
 @import '../../base/module';
+
 .FacultyGrid {
-    position: relative;
-    padding-top: 0;
+  padding-top: 0;
+  background-color: white;
```

Comment:
Nitpick, always best to use a variable here just incase of big theme changes even if it's for white haha